rwengine/src/engine/SaveGame.cpp

```diff
@@ -1313,13 +1313,18 @@ bool SaveGame::getSaveInfo(const std::string& file, BasicState* basicState) {
 std::vector<SaveGameInfo> SaveGame::getAllSaveGameInfo() {
     using namespace boost::filesystem;
 
-    // TODO consider windows
+#if defined(RW_WINDOWS)
+    auto homedir = getenv("USERPROFILE");
+    const char gameDir[] = "\\My Documents\\GTA3 User Files";
+#else
     auto homedir = getenv("HOME");
+    const char gameDir[] = "GTA3 User Files";
```

Comment:
That's very un-posix-ish / un-freedesktop-ish.
We should probably use a proper dot directory for the settings and local program data.